src/App.tsx

```diff
@@ -52,8 +55,23 @@ interface State {
 
 const timeUntilWarn: number = 1000 * 60 * 120;
 const timeFromWarnToLogout: number = 1000 * 60;
+const jwtDecode = require('jwt-decode');
 // const timeoutTotal: number = timeUntilWarn + timeFromWarnToLogout;
 
+function InviteSignupJWT() {
```

Comment:
can you make this an arrow function?

Reply:
I didn't make it an arrow function bc i would always get errors (i think the main one was that hook calls can only be defined inside function components and the [function component with render]([https://medium.com/@will.ulman/routes-and-props-in-react-aec3a65c60d8]) uses an import anyways- at least in the linked example) **BUT i did put the function component in a different file which is then imported into App.tsx.** does this alternative seem ok to u?